src/main/java/io/github/syst3ms/skriptparser/expressions/ExprElement.java

```diff
@@ -0,0 +1,138 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Main;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import io.github.syst3ms.skriptparser.util.math.NumberMath;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * A certain element or multiple elements out of a list of objects.
+ * Remember that in Skript, indices start from 1.
+ *
+ * @name Element
+ * @type EXPRESSION
+ * @pattern ([the] first|[the] last|[a] random|[the] %*integer%(st|nd|rd|th)) element out [of] %objects%
+ * @pattern [the] (first|last) %*integer% elements out [of] %objects%
+ * @pattern %objects%\[%*integer%\]
+ * @since ALPHA
+ * @author Mwexim
+ */
+public class ExprElement implements Expression<Object> {
+
+	static {
+		Main.getMainRegistration().addExpression(
+				ExprElement.class,
+				Object.class,
+				false,
```

Comment:
This should be `true`, otherwise the parser will error straight away if you try to assign it to a single value. If set to true, it will always try to parse the expression, but it will still error if the number ends up not matching in the end.